frontend/packages/helm-plugin/src/components/details-page/HelmReleaseDetails.tsx

```diff
@@ -173,7 +177,7 @@ const HelmReleaseDetails: React.FC<HelmReleaseDetailsProps> = () => {
 
     const getHelmRelease = async () => {
       try {
-        const helmRelease = await fetchHelmRelease(namespace, helmReleaseName);
+        const helmRelease = await fetchHelmRelease(namespace || '', helmReleaseName || '');
```

Comment:
Is there a problem here if either of these values are not actually set? Feels like we shouldn't be making the fetch request in those cases.